src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * 40 - 20;
```

Comment:
You can simplify the conditional logic by returning early. If the condition for the discount is met, return the calculated value immediately without using 'else if' or 'else' blocks.